cmd/ulimit_unix.go

```diff
@@ -44,7 +44,7 @@ func setUlimit(ulimit uint64) error {
 	return syscall.Setrlimit(syscall.RLIMIT_NOFILE, &rLimit)
 }
 
-func GetUlimit() (syscall.Rlimit, error) {
+func getUlimit() (syscall.Rlimit, error) {
```

Comment:
No need for this to be public, and I think we should use the SDKs `limit.GetUlimit()` but will leave that to another PR

Reply:
Done in https://github.com/cloudquery/cloudquery/pull/1048/commits/4ad7eb7f87dd26a0fece3f27b5c24d7339c3611e